foundationdb/datadog_checks/foundationdb/check.py

```diff
@@ -24,9 +29,83 @@ def __init__(self, name, init_config, instances):
         # }
         # self._query_manager = QueryManager(self, self.execute_query, queries=[sample_query])
         # self.check_initializations.append(self._query_manager.compile_queries)
-        pass
 
     def check(self, _):
+        try:
+            status = self.fdb_status_data()
+        except SubprocessOutputEmptyError as e:
+            self.service_check("foundationdb.can_connect", AgentCheck.CRITICAL, message="Did not receive a response from `status json`")
+            raise
+
+        if status[2] != 0:
+            self.service_check("foundationdb.can_connect", AgentCheck.CRITICAL, message="`fdbcli` returned non-zero error code")
+            raise ValueError("Fdbcli failed")
```

Comment:
Probably doesn't want to be captialized, and maybe include the full invocation `fdbcli satus json` in the output.